src/index.ts

```diff
@@ -115,6 +115,11 @@ export default class OpentracingExtension<TContext extends SpanContext>
       childOf: parentSpan || undefined
     });
 
+    // Added to expose bug
+    console.log(
```

Comment:
I added this as an alternative to the snapshot to expose the bug:

For the query
`query {
        a {
          one
          two
        }
      }`

We get the following output

 PASS  src/__tests__/integration-test.ts
  ● Console

      Name: a Parent Name: request
      Name: one Parent Name: a
      Name: a Parent Name: request
      Name: one Parent Name: a
      Name: two Parent Name: *one* . <- This should be a

Reply:
Thank you, this helps understand the issue better.